cmake/Dependencies.cmake

```diff
@@ -2,5 +2,6 @@ TRIBITS_PACKAGE_DEFINE_DEPENDENCIES(
   SUBPACKAGES_DIRS_CLASSIFICATIONS_OPTREQS
     Utils                 packages/Utils                  SS  REQUIRED
     Interface             packages/Interface              SS  REQUIRED
+    Operators             packages/Operators              SS  REQUIRED
```

Comment:
Please ignore all changes related to creating a new subpackage and focus on the actual implementation of the CSRBFs.  We will rebase the PR this as necessary.